config/locales/en.yml

```diff
@@ -188,6 +188,12 @@ en:
 
       <h2 class="govuk-heading-m">If you didn’t expect to see this page</h2>
       <p class="govuk-body">You may be seeing this page because you’ve turned off cookies in your browser. You’ll need to <a class="govuk-link" href="https://ico.org.uk/your-data-matters/online/cookies/" rel="external" target="_blank">turn cookies on</a> before you can use this service.</p>
+  address_api:
```

Comment:
Given these won't be displayed to the user at any point, do we need to store these as an internationalisation?  The English text of a status code can be obtained using (e.g.) `Rack::Utils::HTTP_STATUS_CODES[405]`.

Reply:
Thanks @brucebolt - at the moment, my working assumption is that we might want to display suitable/understandable (content provided) error messages to the user.  If that turns out an incorrect assumption then it makes sense to either use these strings or those provied by the OS Places API.  I would add that OS have changed the meaning of a few of the normal HTTP error codes slightly - so using theirs might be more consistent.